.prettierignore

```diff
@@ -1,4 +1,5 @@
 # Add files here to ignore them from prettier formatting
 /dist
 /coverage
-/.nx/cache
\ No newline at end of file
+/.nx/cache
+__snapshots__
```

Comment:
maybe all snapshots folder?
```suggestion
**/__snapshots__
```

Reply:
Both syntaxes match all nested `__snapshots__` folders. There's no leading `/`, so it doesn't only match root directories.

Reply:
Ah is this a general standard for ignore files?

Reply:
Yes. Prettier uses the [gitignore](https://git-scm.com/docs/gitignore#_pattern_format) syntax.